tests/Prover_tests.js

```diff
@@ -0,0 +1,212 @@
+var should = require("should");
+var rewire = require("rewire");
+
+var createCindy = require("../build/js/Cindy.plain.js");
+
+function FakeCanvas() {
+  this.width = 640;
+  this.height = 480;
+};
+FakeCanvas.prototype.measureText = function(txt) {
+  return { width: 8*txt.length };
+};
+function dummy() { return this; }
+[
+    "addEventListener",
+    "removeEventListener",
+    "arc",
+    "beginPath",
+    "clearRect",
+    "clip",
+    "fill",
+    "getContext",
+    "lineTo",
+    "moveTo",
+    "restore",
+    "save",
+    "stroke",
+    "strokeText",
+    "fillText",
+].forEach(function(m) {
+    FakeCanvas.prototype[m] = dummy;
+});
```

Comment:
I think we should check whether we actually need a canvas, and if so move this fake canvas to a module of its own to be shared between tests. But IIRC I managed to get some tests running without a canvas at all, thanks to the `isNode` setting.